perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py

```diff
@@ -344,12 +345,12 @@ def _perform_testing(dir, test_type, num_samples):
     # Creating config file for mounting with hns enabled.
     with open("/tmp/config.yml",'w') as mount_config:
       mount_config.write("enable-hns: true")
-    mount_flags="--config-file=/tmp/config.yml --stackdriver-export-interval=30s"
+    mount_flags="--config-file=/tmp/config.yml --log-severity=trace --log-format \"text\" --log-file {} --stackdriver-export-interval=30s".format(LOG_FILE)
```

Comment:
What will happen if LOG_FILE has not been set i.e. if --log_file has not been passed ? I am asking because it's `required=False` .

Reply:
Initially, I had set a global variable `LOG_FILE = '/tmp/logfile.txt'`, which would be updated if the log file argument was provided to ensure default log file.
I've updated the implementation so that the log file is now passed as part of another required argument(mount_flags).Thus,marking this as resolved.